src/core/client/stream/tabs/Comments/Stream/FeaturedComments/FeaturedBy.tsx

```diff
@@ -0,0 +1,20 @@
+import { Localized } from "@fluent/react/compat";
+import React, { FunctionComponent } from "react";
+
+import styles from "./FeaturedBy.css";
+
+interface Props {
+  username: string;
+}
+
+const FeaturedBy: FunctionComponent<Props> = ({ username }) => {
+  return (
+    <Localized id="comments-featuredBy" vars={{ username }}>
```

Comment:
I think the `strong` element needs to also be sent through here like `elems={{ strong: <strong className={styles.name} /> }}`. Then it can be used in the localization file like `Featured by <strong>{$username}</strong>`. As is, I'm not seeing the bold applied to the username as expected.

Reply:
just learned something new about Localized!

Reply:
✅